### PR TITLE
Update apollo requirements

### PR DESCRIPTION
**Pull request by zurk**
Signed-off-by: Konstantin Slavnov konstantin@sourced.tech

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ cassandra_driver >= 3.12.0, <4.0
 libMHCUDA >= 2.0, <3.0
 python-igraph >= 0.7, <2.0
 jinja2 >=2.0, <3.0
-#sourcedml >= 0.4.0, <1.0
\ No newline at end of file
+sourced-ml >= 0.4.4, <1.0
```
Comment (vmarkovtsev) [inline requirements.txt]: Should be `<0.5` - we are likely to break API a few more times.

File: setup.py [hunk 2]
```diff
@@ -20,8 +20,8 @@
     install_requires=["cassandra_driver >= 3.12.0, <4.0",
                       "libMHCUDA >= 2.0, <3.0",
                       "jinja2 >=2.0, <3.0",
-                      "python-igraph >= 0.7, <2.0"],
-                      # "sourcedml >= 0.4.0, <1.0"],
+                      "python-igraph >= 0.7, <2.0",
+                      "sourced-ml >= 0.4.4, <1.0"],
```
Comment (vmarkovtsev) [inline setup.py]: Same here

**Comment by vmarkovtsev** [comment]
@zurk According to the logs it misses libsnappy-dev in CI, please fix it here.

**Comment by zurk** [comment]
we have it... https://github.com/src-d/apollo/blob/master/Dockerfile#L17
I am investigating.

**Comment by zurk** [comment]
ping @vmarkovtsev

**Pull request merged**